src/app/db/init_db.py

```diff
@@ -0,0 +1,23 @@
+from app import config as cfg
+from app.api import crud
+from app.db import accesses, users
+from app.api.schemas import AccessCreation, UserCreation
+from app.api.security import hash_password
+
+
+async def init_db():
+
+    login = cfg.SUPERUSER_LOGIN
+
+    # check if access login does not already exist
+    entry = await crud.fetch_one(accesses, [("login", login)])
+    if entry is None:
+
+        hashed_password = await hash_password(cfg.SUPERUSER_PWD)
+
+        access = AccessCreation(login=login, hashed_password=hashed_password, scopes="admin")
```

Comment:
@florianriche quick question: in the swagger, I managed to use admin routes, but it fails on self ID routes. Do we need to add  `"me"` in the scopes?

Reply:
@frgfm I think the proper way would be to add `admin` to the accepted scopes list in the routes.

Reply:
Also, I used the `login/...` route to get my token, but then how can I use GET on `/users/` without receiving a "Not authenticated" in postman? (Swagger is fine, but I don't know if/how the token should be passed to each query)

Reply:
You retrieve the token by querying the login route and then you have to add it in your query headers.
You have a python example  in the PR I just created to deploy the device
https://github.com/pyronear/pyro-api/pull/43/files

```python
class Client():
    def __init__(self):
        self.token = ACCESS_TOKEN
        self.api = API_URL
        self.headers = {"Authorization": f"Bearer {ACCESS_TOKEN}"}

    def hearbeat(self):
        return requests.put(self.api + "/devices/heartbeat", headers=self.headers)
```